### PR TITLE
Add the first test using Testcafe

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Add the first test using Testcafe

**Related issue(s)**

### CODE REVIEW

File: tests/tests/tests.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+test('Namespace `default` card is on the Namespaces list', async t => {
+  const iframe = await getIframe();
+  await t
+    .useRole(adminUser)
+    .switchToIframe(iframe)
+    .expect(Selector('.fd-panel__title').withText('default').exists)
+    .ok();
+});
+
+testIfBackendModuleExists(
```
Comment (sjanota) [inline tests/tests/tests.js]: Maybe, instead of `if` we should just deploy it as separate testdefinition along with application-connector?

Comment (sjanota) [reply tests/tests/tests.js]: Reasoning: backend module is just an implementation detail. What we really want to check is if application-connector is installed it's MF should be available.

File: tests/config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  login: process.env.LOGIN || 'admin@kyma.cx',
+  password: process.env.PASSWORD || 'nimda123',
```
Comment (sjanota) [inline tests/config.js]: The password is no longer `nimda123` by default, so I'd remove the default value

File: tests/helpers.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      .typeText('#password', config.password)
+      .click('#submit-login')
+      .wait(5000);
+  },
+  { preserveUrl: true },
+);
+
+const chooseLoginRole = async t => {
+  try {
+    await Selector('#login').visible; // '.exists' doesn't really wait for the selector..
```
Comment (sjanota) [inline tests/helpers.js]: If it is run on a cluster with multiple dex connectors how long it fails for the selector to fail?

Comment (dariadomagala) [reply tests/helpers.js]: 5 seconds I believe

File: tests/Dockerfile [hunk 4]
```diff
@@ -1,30 +1,10 @@
...
+COPY package*.json ./
 RUN npm install
-
 LABEL source git@github.com:kyma-project/console.git
-
-COPY /ui-tests /app/src
-
-CMD npm run test:cluster
+COPY . .
+RUN apk add curl
```
Comment (Wawrzyn321) [inline tests/Dockerfile:10]: Shouldn't this line be after WORKDIR statement?

Comment (sjanota) [reply tests/Dockerfile:10]: right, it will speed up things a little probably as this layer won't change that much

File: tests/README.md [hunk 5]
```diff
@@ -2,50 +2,34 @@
...
 ## Development
 
-Learn how to run all tests or specific ones.
-
-### Run all tests
-
-> **NOTE:** You need a valid `kubeconfig` issued for a user with admin rights for the cluster. On local deployments, the appropriate `kubeconfig` file is set automatically during Minikube setup. For cluster deployments, you must obtain the `kubeconfig` file manually.
-
-To run all acceptance tests inside a container in the cluster, use this command:
+To run UI tests inside a container in the cluster using Chromium browser, use this command:
```
Comment (alexandra-simeonova) [inline tests/README.md:13]:
```suggestion
To run UI tests inside a container in the cluster using a Chromium browser, use this command:
```

File: tests/README.md [hunk 6]
```diff
@@ -2,50 +2,34 @@
...
 
 ``` bash
 npm run test:cluster
 ```
 
-To run all acceptance tests outside the cluster, use this command:
+To run UI tests inside a container in the cluster using Firefox browser, use this command:
```
Comment (alexandra-simeonova) [inline tests/README.md:19]:
```suggestion
To run UI tests inside a container in the cluster using a Firefox browser, use this command:
```

File: tests/README.md [hunk 7]
```diff
@@ -2,50 +2,34 @@
...
 
-To run all acceptance tests during development against `http://console-dev.kyma.local:4200`, use this command:
+To run UI tests during development against `http://console-dev.kyma.local:4200`, use this command:
 
 ``` bash
 npm run test
 ```
 
-> **NOTE:** In the second case, take note of which **GraphQL API** endpoint you use.
+To run UI tests during development against `http://console-dev.kyma.local:4200` using headless browser, use this command:
```
Comment (alexandra-simeonova) [inline tests/README.md:31]:
```suggestion
To run UI tests during development against `http://console-dev.kyma.local:4200` using a headless browser, use this command:
```

**Pull request merged**